spec/models/post_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
```

Comment:
Please add unit tests for  `#update_users_posts_counter` in Posts model, `#update_comments_posts_counter` in Comments model, and `#update_posts_likes_counter` in Likes model. It's a requirement to test custom methods in models.